modules/mono/editor/GodotTools/GodotTools.ProjectEditor/default.csproj.template

```diff
@@ -0,0 +1,67 @@
+<Project Sdk="Godot.NET.Sdk/%GODOT_NET_SDK%">
+  <PropertyGroup>
+    <RootNamespace>%ROOT_NAMESPACE%</RootNamespace>
+  </PropertyGroup>
+  <PropertyGroup Condition="$(RuntimeIdentifier.Contains('ios'))">
+    <TargetFramework>net8.0</TargetFramework>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="!$(RuntimeIdentifier.Contains('ios'))">
+    <TargetFramework>net6.0</TargetFramework>
+    <EnableDynamicLoading>true</EnableDynamicLoading>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="$(RuntimeIdentifier.Contains('ios'))">
+    <PublishAot>true</PublishAot>
+    <PublishAotUsingRuntimePack>true</PublishAotUsingRuntimePack>
+    <RunAOTCompilation>true</RunAOTCompilation>
+    <ForceFullAOT>true</ForceFullAOT>
+    <SelfContained>true</SelfContained>
+    <GenerateXcodeProject>true</GenerateXcodeProject>
+    <UseNativeAOTRuntime Condition="'$(UseNativeAOTRuntime)' == ''">true</UseNativeAOTRuntime>
+    <Optimize>false</Optimize>
+    <NativeDebugSymbols>true</NativeDebugSymbols>
```

Comment:
Is this needed? This should default to true if DebugType != '' or != 'none'. If DebugType is one of those, the C# compilation is not generating debug info and native debugging symbols won't have much for this project anyway.

Reply:
Not needed, another artifact of my copy-paste.